app/Http/Requests/ProgramRequest.php

```diff
@@ -23,6 +23,13 @@ public function authorize()
      *
      * @return array
      */
+    public function rules() 
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 26</br>Whitespace found at end of line